lib/Model/CMakeLists.txt

```diff
@@ -58,6 +58,7 @@ tuple_tree_generator(
   GENERATED_HEADERS
   GENERATED_IMPLS
   "${JSONSCHEMA_PATH}"
+  "${CMAKE_BINARY_DIR}/lib/python/${PYTHON_GENERATED_MODEL_PATH}"
```

Comment:
Wouldn't it be better to push `${CMAKE_BINARY_DIR}/lib/python/` within `PYTHON_GENERATED_MODEL_PATH`?
Otherwise it has a strange value.

Reply:
Why not?
Maybe he is the special kid and you want to compute the relative path there?
Otherwise `PYTHON_GENERATED_MODEL_PATH` is quite misleading.

Reply:
`python_module` assumes that the paths it is passed have to be installed in `${CMAKE_BINARY_DIR}/lib/python`, I tried to refactor it and anything I came up with is worse than this